src/components/QuizGame/index.jsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from "react";
+import { fetchQuizQuestions } from "../../components/QuizGameApi";
+import QuestionCard from "../../components/QuizGameQuestions";
+import "./index.css";
+const Difficulty = {
+  EASY: "easy",
+  MEDIUM: "medium",
+  HARD: "hard",
+};
```

Comment:
should be `DIFFICULTY` and a newline needed after imports

Reply:
PTAL